controllers/storageclassrequest/storageclassrequest_controller.go

```diff
@@ -45,6 +45,11 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
+const (
+	storageClassRequestFinalizer = "ocs.openshift.io.storageclassrequest"
+	forceDeletionAnnotationKey   = "ceph.rook.io/force-deletion"
```

Comment:
Later, It got changed to **rook.io/force-deletion**

Reply:
thanks for the updated, raised https://github.com/red-hat-storage/ocs-operator/pull/2573